.pre-commit-config.yaml

```diff
@@ -44,6 +44,7 @@ repos:
     rev: 3.9.2
     hooks:
       - id: flake8
+        additional_dependencies: [flake8-typing-imports==1.10.1]
```

Comment:
Can you add `flake8-typing-imports==1.10.1` to `requirements_text_pre_commit.txt`, too?